crates/astria-composer/src/searcher/rollup.rs

```diff
@@ -53,17 +53,17 @@ impl Rollup {
         // Note that this will panic on invalid indices. However, these
         // accesses will always be correct because the regex will only
         // match when these capture groups match.
-        let chain_id = caps["chain_id"].to_string().to_lowercase();
+        let rollup_name = caps["rollup_name"].to_string().to_lowercase();
         let url = caps["url"].to_string();
         Ok(Self {
-            chain_id,
+            rollup_name,
             url,
         })
     }
 
     pub(super) fn into_parts(self) -> (String, String) {
         let Self {
-            chain_id,
+            rollup_name: chain_id,
             url,
         } = self;
         (chain_id, url)
```

Comment:
And with the change above:
```suggestion
        (rollup_name, url)
```